daemon/mgr/container.go

```diff
@@ -632,6 +616,19 @@ func (mgr *ContainerManager) stoppedAndRelease(id string, m *ctrd.Message) error
 	return nil
 }
 
+func (mgr *ContainerManager) stoppedAndRelease(id string, m *ctrd.Message) error {
```

Comment:
Could you change a more readable function name?
Actually I cannot get the actual meaning of `stoppedAndRelease`. Why to use `stopped`?
Or we can add more function comments above the function.